rust/bridge/node/bin/gen_ts_decl.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
This is based on `gen_java_decl.py`. Do you think it's worth me putting in effort to factor out a shared infrastructure?

Reply:
IMO lets not overthink it at this stage. We can get Desktop onto Rust with everything supported and then think about merging scripts and other cleanups.

It's a different problem from the general issue of the bridge - there the effort of having to support multiple bridges increases as we increase the number of bridged symbols, while these scripts only need to change when we need to support a new in/out type which is much less common.